src/AdamStipak/Webpay/PaymentRequest.php

```diff
@@ -79,4 +92,47 @@ public function setMerchantNumber ($number) {
   public function setDescription($value){
     $this->params['DESCRIPTION'] = $value;
   }
+
+  /**
+   * Set preferred payment method
+   *
+   * @param string $method
+   */
+  public function setPayMethod(string $method): void {
+    if($this->isValidPayMethod($method)) {
+      $this->params['PAYMETHOD'] = $method;
+    }
```

Comment:
What about invalid value? Is there some default in GPwebpay? If not there are options:
- add exception if method is not valid
- use some default method